frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts

```diff
@@ -1,7 +1,7 @@
 import { K8sResourceKind } from '@console/internal/module/k8s';
 
 export const minSelectedNode = 3;
-export const taintObj = {
+export const TAINT_OBJ = {
```

Comment:
Not new, but you should give this a more specific name like `ocsTaint`. Consider `Object.freeze` to avoid accidentally modifying the value.

```suggestion
export const ocsTaint: Taint  = Object.freeze({
```

Reply:
Ack. Making the constant defined in capital letters as the same pattern is being followed across dashboards.

Reply:
> Ack. Making the constant defined in capital letters as the same pattern is being followed across dashboards.

Objects are a little different than strings and numbers, and this doesn't match the naming of the object just below.